lib/team_statistics.rb

```diff
@@ -195,6 +204,4 @@ def rival(passed_id)
     rival = @win_ratio_by_opponent.invert.min[1]
     @team_name_by_id[rival]
   end
-
-
 end
```

Comment:
Great job. Can't wait to refactor tomorrow to get under 150

Reply:
YES! In the PR details we have some suggestions for refactoring we can go through on a new branch